package.json

```diff
@@ -83,30 +83,31 @@
     "webpack": "^4.41.0"
   },
   "peerDependencies": {
-    "i18next": "^17.0.17",
-    "i18next-browser-languagedetector": "^3.0.3",
-    "i18next-icu": "^1.1.2",
-    "i18next-xhr-backend": "^3.2.0",
+    "i18next": "^19.1.0",
+    "i18next-browser-languagedetector": "^4.0.1",
+    "i18next-icu": "^1.3.1",
+    "i18next-http-backend": "^1.0.15",
     "ipfs-css": "^0.13.1",
     "prop-types": "^15.7.2",
-    "react": "^16.10.1",
-    "react-dom": "^16.10.1",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
     "react-helmet": "^5.2.1",
-    "react-i18next": "^11.2.2",
+    "react-i18next": "^11.2.7",
     "react-loadable": "^5.5.0",
     "react-virtualized": "^9.21.1",
     "redux-bundler": "^26.0.0",
     "redux-bundler-react": "^1.2.0",
     "tachyons": "^4.11.1"
   },
   "engines": {
-    "node": ">=8",
-    "npm": ">=5"
+    "node": ">=10",
+    "npm": ">=6"
   },
   "browsers": [
-    ">0.25%",
-    "not ie 11",
-    "not op_mini all"
+    ">1%",
+    "last 2 versions",
+    "Firefox ESR",
+    "not ie < 11"
```

Comment:
Switched to what aegir uses: https://github.com/ipfs/aegir/blob/v22.0.0/package.json#L142-L146